pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -138,6 +138,8 @@ A specific solver should only be configured if you want to force preCICE to use
   - Removed callback functions `vertexCallback` and `postAction` from `PythonAction` interface.
   - Removed timewindowsize from the `performAction` signature of `PythonAction`. The new signature is `performAction(time, data)`
 
+- We removed the Broyden acceleration. Use IMVJ for Broyden-like acceleration instead.
```

Comment:
What does "Broyden-like" mean? Can we cite some reference here?

Reply:
I extended the comment. Understandable like this?